src/style.css

```diff
@@ -1 +1,25 @@
 /* add styles here */
+body {
+  margin: 0;
+}
```

Comment:
It's recommended to avoid styling tags directly (except for `html` and `body`). However, since this is the body tag and it's being used to reset default margin, this is acceptable.